docs/queryworkers/attribute-based-access.md

```diff
@@ -0,0 +1,260 @@
+---
+sidebar_position: 40
+title: Attribute-Based Access Control
+---
+
+For this example, assume the following organizations each have employees who must access their own data:
+
+- Supplier Store
+- Partner 1
+- Partner 2
+
+Each partner store needs an API key with appropriate attributes to limit their access. Additionally, individual supplier employees need similar limitations.
+
+For this example, configure an API key for each of the following users, then add an attribute with these values:
+
+- Partner 1:
+    - **Attribute -** `partner`
+    - **Value -** `partner1`
+- Partner 2:
+    - **Attribute -** `partner`
+    - **Value -** `partner2`
+- Supplier Admin:
+    - **Attribute -** `employee`
+    - **Value -** `admin`
+- Supplier Staff:
+    - **Attribute -** `employee`
+    - **Value -** `staff`
+
+Refer to the [Add Attributes to API Keys](../account-management/attributes/add-attributes-api.md) section for instructions on adding attributes to API keys.
+
+Create two collections which will contain customer data for the examples: `item` and `customer`.
+
+Add the following data to the `item` collection:
```

Comment:
I recommend actually making this a query that will add the info to the collections. You can get the syntax from the Game of Thrones tutorial, now called Query Tutorial in the Queries section. Just makes life a tad easier for the users. Don't forget to test the queries!